.github/workflows/test-changed-firestore.yml

```diff
@@ -16,7 +16,7 @@ name: Test Firestore
 
 on:
   workflow_dispatch:
-  pull_request:
```

Comment:
Why this change?

Reply:
I made this change so that I can test the workflow on my branch. I'll revert it